BDArmory/Weapons/Missiles/MissileLauncher.cs

```diff
@@ -1897,6 +1952,12 @@ IEnumerator CruiseRoutine()
                             gpe.Current.emit = false;
                         }
                     }
+                //Thrust
+                if (useFuel && burnRate > 0 && burnedMass < massToBurn)
+                {
+                    burnedMass = Mathf.Min(burnedMass + burnRate, massToBurn);
+                    ordinanceMass = -burnedMass;
```

Comment:
Again, you're running Mathf.Min on a positive number and a positive number + another positive number, this is only ever going to return burnedMass. burnedMass has also been defined on 1897 to be the same as ordinanceMass, so 
like with its sibling in BoostRoutine(), this doesn't need to be more than 
```
if (useFuel && burnRate > 0 && -ordinancemass < massToBurn)
      ordinancemass -= burnRate;
```

Reply:
i'm using burned mass because i think it make it more readable, but i can change that.

Reply:
Yeah, I think `burnedFuelMass` makes a better variable name than `ordinanceMass`. I've made this change (replacing `-ordinanceMass` with `burnedFuelMass`) and a few other minor adjustments to the log messages and pushed the result to a new pr517 branch https://github.com/BrettRyland/BDArmory/compare/dev...BrettRyland:BDArmory:pr517?expand=1#files_bucket (ignore the extra changelog stuff, I started making other adjustments to it while on the wrong branch).

Also, I've only just noticed that you made this PR against the master branch. Normally PRs should be made against the dev branch so that they get closed once they're merged into dev instead of waiting for a release from master. Github lets you change the target branch, but warns that some commits and comments may get removed (though I'm not sure how much and I'm planning on making a new release once this gets merged anyway), so don't worry about changing it now, just remember to make the PRs against dev in the future.

Reply:
@BrettRyland I'm not seeing any other immediate outstanding issues; I think we can go ahead an merge it.

Reply:
OK, it's now merged to dev. I'll do a release later tonight after I've had dinner.